Source/SmallBasic.Editor/Libraries/Graphics/TurtleGraphicsObject.cs

```diff
@@ -22,6 +22,8 @@ public override void ComposeTree(TreeComposer composer)
                 attributes: new Dictionary<string, string>
                 {
                     { "href", $"Turtle.svg" },
+                    { "height", "61" },
+                    { "width", "48" }
```

Comment:
Instead of duplicating these values from [this](https://github.com/sb/smallbasic-editor/blob/master/Source/SmallBasic.Editor/Libraries/Shapes/TurtleShape.cs#L17-L19), I suggest following [the example](https://github.com/sb/smallbasic-editor/blob/master/Source/SmallBasic.Editor/Libraries/Shapes/EllipseShape.cs#L17-L19) of `EllipseShape.cs` by lowering these properties from `TurtleShape.cs` to `TurtleGraphicsObject.cs`, and having the outer shape return the values from the inner graphics object.

Reply:
I made an update according to your suggestions.